### DISCUSSION TITLE
Climate mitigation & policy

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#293] [General]
[FROM BASECAMP]

**Kate Longley-Wood**
Hi Clara and team -- I'm putting the mock-up and the draft tabular data that goes along with it below. Note that I'm still waiting to acquire/verify the data for some of these fields, but at present, this should give you an idea of the general structure. A few challenges: 

1. You'll see this app involves a lot of stats/information "bullets". We're looking for design advice from the team to help make this information easier analyze -- maybe organized by theme? The first 4 stats, the next 2, and the last 3 could potentially be lumped

2. It would be great to have some sort of graphic in here, we're just not sure what's best. We had some initial thoughts from our team members but I'm still waiting for more information from them to flesh out these ideas, so if you have any thoughts, please send them our way. 

[GMW_Climate_Policy_Mockup_20210701.pptx](https://github.com/Vizzuality/mangrove-atlas/files/6776494/GMW_Climate_Policy_Mockup_20210701.pptx)
[Climate_Policy_Tool_Data_20210701_KLW.csv](https://github.com/Vizzuality/mangrove-atlas/files/6776495/Climate_Policy_Tool_Data_20210701_KLW.csv)

**Comment by dhakelila** [comment]
[Basecamp original message](https://basecamp.com/1756858/projects/16449126/messages/96909250)

**Comment by dhakelila** [comment]
@klongleywood, you mentioned in your previous message that your team has some ideas to represent these values. 
We've been discussing this internally and I think it would be good to chat all together. 

If you have more ideas from your team, please, send. 

If not, we'd like to discuss this during our next weekly call on Tuesday. Dani and Tamara will join us at the beginning of the meeting for this purpose. Feel free to invite key people from your team as well.

**Comment by dhakelila** [comment]
Notes from the 13th July meeting: 

- Vizzuality presented a proposal that needs to be approved but GMA. ([Shared on Github for your evaluation](https://github.com/Vizzuality/mangrove-atlas/discussions/285))
- This widget is now out of the scope for the 26th of July due to time constraints.

Pending: 
- [GMA feedback](https://github.com/Vizzuality/mangrove-atlas/issues/291). 
- [GMA final data](https://github.com/Vizzuality/mangrove-atlas/issues/292).

**Comment by lyubovpencheva** [comment]
This is an old one.
